core-services/prow/02_config/_plugins.yaml

```diff
@@ -256,27 +256,6 @@ cherry_pick_unapproved:
   comment: This PR is not for the master branch but does not have the `cherry-pick-approved`  label.
     Adding the `do-not-merge/cherry-pick-not-approved`  label.
 config_updater:
-  cluster_groups:
-    build_farm_ci:
-      clusters:
-      - app.ci
-      - build01
-      - build02
-      - build03
-      - build04
-      - vsphere
-      namespaces:
-      - ci
-    build_farm_ocp:
-      clusters:
-      - app.ci
-      - build01
-      - build02
-      - build03
-      - build04
-      - vsphere
-      namespaces:
-      - ocp
```

Comment:
the changes to this file don't seem related to this PR, are they deliberate?  was their a tooling change and this is just the first PR since the tooling change?

@petr-muller

Reply:
I am not aware of any tool changes that would cause this. I agree that I would not expect this change to be there. How was this PR created?

Reply:
This PR was created by `make new-repo`. I did not manually make any changes under `core-services/prow/02_config`. They were automatically done by the script.

Reply:
I can try remove changes under `core-services/prow/02_config` and see if tests can still pass.

Reply:
Thanks, I'll try to reproduce and file that as a bug. Please remove the changes in this file from this PR.

Reply:
https://issues.redhat.com/browse/DPTP-2721

Reply:
Test will not pass if you do that. The unwanted change is just in `core-services/prow/02_config/_plugins.yaml` file, the `_pluginconfig.yaml` that was originally present in a directory structure under `core-services/prow/02_config` was actually needed.

Reply:
Done. Removed just the `core-services/prow/02_config/_plugins.yaml` as suggested.